src/api/modelTypes.js

```diff
@@ -350,19 +350,16 @@ export type Reaction = $ReadOnly<{|
 export type MessageSnapshot = $ReadOnly<{|
   user_id: number,
   timestamp: number,
+  topic: string,
+  content: string,
+  rendered_content: string,
 
-  /** Docs unclear but suggest absent if only content edited. */
-  topic?: string,
```

Comment:
That discussion here:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/get-message-history.20docs/near/921587

Whenever we find a place where the API docs seem to be wrong, I always want to discuss it in chat and bring it to Tim's attention there. Reasons include:
* I want the API docs to converge on being right too :slightly_smiling_face: -- if we have a steady state where our types say one thing and the API docs say another, that'll be confusing to a later reader
* If for whatever reason the API docs don't get updated promptly, I'll want the comments in our code to be super explicit that the API docs say X but the truth is Y, and here's how we've determined the truth is Y. That gives the future reader a hope of not being confused by the contradiction, because at least they can see that the author of the types was aware of what the API docs said, and wasn't just misreading them, or working from what at that future time is an out-of-date version
* The answer might be that the documented API is as intended, and the server's behavior that disagrees is a bug!

As of right now (since a few minutes ago), the status is that Tim confirms the current behavior is what you're seeing empirically -- but potentially wants to fix it by changing the behavior to match the docs instead of vice versa.